manifests/ocs/storageclass.yaml

```diff
@@ -0,0 +1,22 @@
+---
+apiVersion: ceph.rook.io/v1
+kind: CephBlockPool
+metadata:
+  name: openshift-storage
+  namespace: openshift-storage
+spec:
+  replicated:
+    size: 3
+---
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  annotations:
+    storageclass.kubernetes.io/is-default-class: "true"
+  name: rook-ceph-block
+provisioner: ceph.rook.io/block
```

Comment:
We'll want to use a csi-based storage class instead of the rook flex driver. 
- For an example of the storage class, see [here](https://github.com/rook/rook/blob/master/cluster/examples/kubernetes/ceph/csi/example/rbd/storageclass.yaml)
- I'd suggest waiting until this PR is merged since it will be much simpler to use csi in rook: https://github.com/rook/rook/pull/3562. In the meantime you could certainly use the flex-based storage class.